File: javascript-closure-angular-client/API/Client/PurchaseTransactionIn.js
goog.provide('API.Client.PurchaseTransactionIn');

/**
 * @record
 */
API.Client.PurchaseTransactionIn = function() {}

/**
 * @type {!API.Client.PurchaseHeaderIn}
 * @export
 */
API.Client.PurchaseTransactionIn.prototype.header;

/**
 * @type {!Array<!API.Client.PurchaseLinesIn>}
 * @export
 */
API.Client.PurchaseTransactionIn.prototype.lines;

File: dot-net-client/src/Avalara.AvaTax.Brazil.RestClient/Model/PurchaseTransactionIn.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Avalara.AvaTax.Brazil.RestClient.Model
{
    /// <summary>
    /// PurchaseTransactionIn
    /// </summary>
    [DataContract]
    public class PurchaseTransactionIn
    {
        public PurchaseTransactionIn()
        {
            Header = new PurchaseHeaderIn();
            Lines = new List<PurchaseLinesIn>();
        }

        /// <summary>
        /// Gets or Sets Header
        /// </summary>
        [DataMember(Name = "header", EmitDefaultValue = false, IsRequired = true)]
        public PurchaseHeaderIn Header { get; set; }

        /// <summary>
        /// Gets or Sets Lines
        /// </summary>
        [DataMember(Name = "lines", EmitDefaultValue = false, IsRequired = true)]
        public List<PurchaseLinesIn> Lines { get; set; }
    }
}
